spdx-utils/src/main/kotlin/SpdxUtils.kt

```diff
@@ -50,16 +48,21 @@ val NON_LICENSE_FILENAMES = listOf(
     "META-INF/DEPENDENCIES"
 )
 
+/**
+ * Return a string of hexadecimal digits representing the bytes in the array.
+ */
+private fun ByteArray.toHexString(): String = joinToString("") { String.format("%02x", it) }
```

Comment:
The title and body of the commit message talk of two different changes, suggesting it should have been two commits instead. Also, the change in `SpdxUtilsTest` seems unrelated.

Reply:
I've split out the `SpdxUtilsTest` change, but the `toHexString()` change *is* actually part of the topic as explained in the commit message:

> toHexString() is copied from the utils module as we do not want spdx-utils to depend on it, so that spdx-utils could be published independently.

`toHexString()` replaces `Hex.encodeHexString()` from `commons-codec`.